project_service/app.js

```diff
@@ -432,7 +435,9 @@ async function initializeNode0(url) {
     }
     // Launch headless chrome
     // TODO Create a dedicated user to run headless chrome so that sandbox args can be removed and security improved (see here: https://github.com/GoogleChromeLabs/lighthousebot/blob/master/builder/Dockerfile#L35-L40)
-    let browser = await puppeteer.launch({args: ['--no-sandbox', '--disable-setuid-sandbox'], dumpio: RANK_0_DEBUG});
+
+    let browser = await puppeteer.launch({args: ['--no-sandbox', '--disable-setuid-sandbox','--remote-debugging-port=9222'], dumpio: RANK_0_DEBUG});
```

Comment:
Is this okay to have in prod? I'm doubtful we want remote debugging enabled?

CC @zacharysang

Reply:
Hmm yeah. Should probably put this behind the IS_DEV flag. [Here](https://github.com/zacharysang/flock/blob/master/project_service/app.js#L418) is an example where we use the localtunnel url in the dev environment, but use the master's `FLOCK_URL` variable outside of dev